integration_tests/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
```

Comment:
I updated the cmake minimum version here. `cmake` with version `3.5`  is unable to find `python` from the `conda` environment (It always picks the system level `python`). It seems `cmake` with version `15` and above supports finding python from `conda` environment (reference https://gitlab.kitware.com/cmake/cmake/-/issues/21322#note_845923). `Python` from conda environment is needed as the conda environment contains the installed numpy package.